commit cb1172558bc25c2c6fd33e7c39c9b746c26196b6
Author: q66 <q66@chimera-linux.org>
Date:   Thu Oct 15 18:13:40 2026 +0200

    disable networking check for riscv64
    
    since 158, this fails in the rust static binary, and i cba to find
    out why

diff --git a/config/makefiles/rust.mk b/config/makefiles/rust.mk
index ab33044..c291fa0 100644
--- a/config/makefiles/rust.mk
+++ b/config/makefiles/rust.mk
@@ -327,6 +327,7 @@ force-cargo-library-build:
 # The check only works when rust code is built with -Clto but without MOZ_LTO_RUST_CROSS.
 # Sanitizers and sancov also fail because compiler-rt hooks network functions.
 ifndef MOZ_PROFILE_GENERATE
+ifeq (,$(findstring riscv64,$(RUST_TARGET)))
 ifeq ($(OS_ARCH), Linux)
 ifeq (,$(RUST_SANCOV_FLAGS)$(MOZ_ASAN)$(MOZ_TSAN)$(MOZ_UBSAN))
 ifndef MOZ_LTO_RUST_CROSS
@@ -337,6 +338,7 @@ endif
 endif
 endif
 endif
+endif
 
 $(eval $(call make_cargo_rule,$(RUST_LIBRARY_FILE),force-cargo-library-build))
 
